e2e/scenario/vm.go

```diff
@@ -0,0 +1,7 @@
+package scenario
+
+var DefaultGPUSeriesVMSizes = map[string]string{
```

Comment:
I decided to try going the `go:embed` route for the default SIG image versions map, wondering if we should also do the same here - can potential follow-up on that in a separate PR as well

Reply:
deferring for now